src/utilities/util_printlog.c

```diff
@@ -62,6 +67,9 @@ util_printlog(WT_SESSION *session, int argc, char *argv[])
         case 'm': /* messages only */
             LF_SET(WT_TXN_PRINTLOG_MSG);
             break;
+        case 'u': /* print user data, don't redact */
+            LF_CLR(WT_TXN_PRINTLOG_REDACT);
+            break;
```

Comment:
I know how we got here, but it feels a little odd to set the default state and then clear it if a command line is specified, that is, I'd normally expect the default state to be the default, and command-line flags would add to that state. In other words, the flag would be "WT_TXN_PRINTLOG_SHOW_USER_DATA", not "WT_TXN_PRINTLOG_REDACT", and we'd set the flag here, rather than both setting and clearing the flag.

Another argument would be that if someone writes their own printlog utility, or incorporates it into their program, the default is to show user data, not to hide it (that is, the WiredTiger default API is to show user data), and it feels odd for the WT API to be different from the wt printlog API.

I wouldn't fight hard for this, but it feels cleaner.

Reply:
I can agree with the sentiment. I'm going to call it `WT_TXN_PRINTLOG_UNREDACT`. It generalizes it from user data to whatever we choose to redact.

Reply:
Yes, that's a better name/behavior -- thank you!